proto/api/read_state.proto

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+// To compile the proto, run:
+//      protoc -I. -I ./../types --go_out=plugins=grpc:$GOPATH/src *.proto
+syntax = "proto3";
+package iotexapi;
+option go_package = "github.com/iotexproject/iotex-proto/golang/iotexapi";
+
+
+message ReadStakingDataMethod {
+	enum Name {
+		INVALID = 0;
+		GET_BUCKETS = 1;
+		GET_BUCKETS_BY_VOTER = 2;
+		GET_BUCKETS_BY_CANDIDATE = 3;
+		GET_CANDIDATE = 4;
+		GET_CANDIDATE_BY_NAME = 5;
+	}
+	Name method = 1;
+}
```

Comment:
as discussed yesterday, this is actually redundant, as method could be tell by the "ReadStakingDataRequest". Should we just leave the method name as empty for staking protocol?

Reply:
i think that works too. but it is a little bit hard to communicate that out. I agree this is redundant, but people understand what to fill once they read it, instead of asking question about what to fill in method parameter ?